### ISSUE TITLE
fix references to powers of ten

### ISSUE THREAD

**Issue by markburns** [markburns/numbers_in_words#7]
see https://github.com/markburns/numbers_in_words/pull/6/files#r46365729

**Comment by dimidd** [comment]
Right, it should be language dependent, e.g. some languages have a word for 10000, while English doesn't. How do I access the language inside the `NumberParser` module?

**Comment by markburns** [comment]
Actually, I think we should wait until we have one more language before thinking about this: once #12 is done it will be clearer which code belongs where. Otherwise we're guessing too much upfront. (Which is what I had already done when initially writing this stuff anyway).
